wdh_server/templates/articles.html

```diff
@@ -5,27 +5,30 @@
 <ul>
 	{% for article in articles %}
 	<li>
-		<a href="#article{{ article.id }}" rel="http://rels.example.org/article"></a>
-		<article id="article{{ article.id }}">
-			<h1>
-				<a href="{{ url_for("article", article_id=article.id) }}" rel="self">
-					{{ article.title }}
-				</a>
-			</h1>
-			<dl>
-				<dt>title</dt>
-				<dd>{{ article.title }}</dd>
+        <details>
+            <summary>
+                <h1>
+		            <a href="{{ url_for("article", article_id=article.id) }}" rel="http://rels.example.org/article">
+                        {{ article.title }}
+                    </a>
+                </h1>
```

Comment:
the `h1` belongs into `article`, as  - see http://html5doctor.com/outlines/#sectioning-elements

we had some overlap here (my bad), so I'd already changed this template (though not others) in 266cc6f6a683491eec173e6f0b19e2e2c8e99252